core/src/io/anuke/mindustry/ui/fragments/HudFragment.java

```diff
@@ -249,8 +250,15 @@ public void build(Group parent){
             }).top().left();
         });
 
+        Table map_table = new Table();
         //minimap
-        parent.fill(t -> t.top().right().add(new Minimap()).visible(() -> Core.settings.getBool("minimap") && !state.rules.tutorial));
+        map_table.add(new Minimap().visible(() -> Core.settings.getBool("minimap") && !state.rules.tutorial));
+        map_table.row();
+        //position
+        map_table.label(() -> String.format("%.0f,%.0f", Vars.player.getX(), Vars.player.getY())).visible(
```

Comment:
Use player.x/y instead of the getters, and don't use `String.format`, as that is very slow. Instead, just do `(int)player.x + "." + (int)player.y`.